### ISSUE TITLE
Add Highpass Filter @ 5Hz

### ISSUE THREAD

**Issue by sneakernets** [Wohlstand/libADLMIDI#21]
Without a highpass filter:
![image](https://user-images.githubusercontent.com/5595293/30667590-abc7b5be-9e1d-11e7-842d-784e9f89fc91.png)
This is also splattering like mad. It's basically brickwalled.

With a highpass filter:
![image](https://user-images.githubusercontent.com/5595293/30667599-b4775458-9e1d-11e7-9ee2-88e18c128604.png)

That's much better.

Seems like it needs to be done on each chip's output, if that's even possible.

**Comment by Wohlstand** [comment]
Try to capture real chip's output and tell me does it produces same/similar?

> Seems like it needs to be done on each chip's output, if that's even possible.

That possible, but, still a question how result will be heard

**Comment by sneakernets** [comment]
Unfortunately, I do see this on the real chip, but it was never an issue until multiple cards came into question. I managed to "fix" it by reducing the volume of the offending instruments. Heretic uses a lot of percussion that caused the splattering.

**Comment by jpcima** [comment]
This is easily implemented externally on software using the libADLMIDI, like I do in all of them.
If not implemented in libADLMIDI, at least maybe midiplay can have it?

**Comment by Wohlstand** [comment]
Yeah, let's try to make this on MIDIPlay side and let it be toggle-able to be able compare results :fox_face: :+1:

**Comment by jpcima** [comment]
Oh yeah.. but I forgot about how audio is not so easily manipulated in this, because of having all of the possible format choices.
Do you want that I edit adljack to have the hpf switchable?
